sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/DataCollection/DocumentHelper.cs

```diff
@@ -19,7 +21,87 @@ internal static class DocumentHelper
     {
         // TODO: NEED TO HANDLE UNIQUE MAXLENGTH VALUES FOR DOCUMENT TYPES. SEE SWAGGER FOR MAXLENGTH VALUES.
 
-        internal static RemoteDependency ConvertToRemoteDependency(Activity activity)
+        #region Document Buffer Extension Methods
+        public static void AddExceptionDocument(this DoubleBuffer buffer, ActivityEvent activityEvent)
+        {
+            try
+            {
+                var exceptionDocument = ConvertToExceptionDocument(activityEvent);
+                buffer.WriteDocument(exceptionDocument);
+            }
+            catch (System.Exception ex)
+            {
+                LiveMetricsExporterEventSource.Log.FailedToCreateTelemetryDocument("ExceptionDocument", ex);
+            }
+        }
+
+        public static void AddExceptionDocument(this DoubleBuffer buffer, System.Exception exception)
```

Comment:
Where this method is used?

Reply:
This will be used in the LogRecordProcessor, which I'm planning to introduce in my next PR.

I added in this PR because I wanted to validate my overall design. 
With this change, I'm not expecting any more large changes to the DocumentHelper class.

I can remove it to reduce the scope of this PR if you would prefer.